### PR TITLE
[eas-cli] Removes "please" from output

### PR DESCRIPTION
**Pull request by jonsamp**
# Why

We want to [remove the word "please"](https://www.youtube.com/watch?v=FvNccZxovAM&ab_channel=maikmu) from the output of our commands. This PR replaces instances of "please" with either by removing "please" or by slightly re-wording the phrase to no longer include "please".

# Test Plan

Make sure that the output still reads well and instructs developers correctly.

### CODE REVIEW

File: packages/eas-cli/src/commands/branch/delete.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default class BranchDelete extends EasCommand {
       ({ name } = await promptAsync({
         type: 'text',
         name: 'name',
-        message: 'Please enter the name of the branch to delete:',
+        message: 'Enter the name of the branch to delete:',
```
Comment (kbrandwijk) [inline packages/eas-cli/src/commands/branch/delete.ts:118]: Can we change these, and the one(s) below, to 'Provide' as well for consistency. Compare:

'Provide the name of the channel to delete:'

File: packages/eas-cli/src/commands/branch/rename.ts [hunk 2]
```diff
@@ -90,7 +90,7 @@ export default class BranchRename extends EasCommand {
       ({ currentName } = await promptAsync({
         type: 'text',
         name: 'currentName',
-        message: 'Please enter the current name of the branch to rename:',
+        message: "Enter the name of the branch you'd like to rename:",
```
Comment (kbrandwijk) [inline packages/eas-cli/src/commands/branch/rename.ts:93]: Provide...

File: packages/eas-cli/src/commands/update/list.ts [hunk 3]
```diff
@@ -73,7 +73,7 @@ export default class BranchView extends EasCommand {
         ({ name: branchInteractive } = await promptAsync({
           type: 'text',
           name: 'name',
-          message: 'Please enter the name of the branch whose updates you wish to view:',
+          message: 'Enter the name of the branch whose updates you wish to view:',
```
Comment (kbrandwijk) [inline packages/eas-cli/src/commands/update/list.ts:76]: Provide...

File: packages/eas-cli/src/credentials/ios/actions/SetUpInternalProvisioningProfile.ts [hunk 4]
```diff
@@ -70,15 +70,15 @@ export class SetUpInternalProvisioningProfile {
...
+          `You're in non-interactive mode. You have set up both adhoc and universal distribution credentials. Set the 'enterpriseProvisioning' property (to 'adhoc' or 'universal') in eas.json to choose the credentials to use.`
         );
       } else if (adhocBuildCredentialsExist) {
         return await this.setupAdhocProvisioningProfileAsync(ctx);
       } else if (enterpriseBuildCredentialsExist) {
         return await this.setupUniversalProvisioningProfileAsync(ctx);
       } else {
         throw new Error(
-          `You're in non-interactive mode. EAS CLI couldn't find any credentials suitable for internal distribution. Please run again in interactive mode.`
+          `You're in non-interactive mode. EAS CLI couldn't find any credentials suitable for internal distribution. Run again in interactive mode.`
```
Comment (kbrandwijk) [inline packages/eas-cli/src/credentials/ios/actions/SetUpInternalProvisioningProfile.ts:81]: In other places, we use 'Run _this command_ again in interactive mode'

File: packages/eas-cli/src/build/utils/printBuildInfo.ts [hunk 5]
```diff
@@ -99,14 +99,14 @@ export function printDeprecationWarnings(deprecationInfo?: EasBuildDeprecationIn
     return;
   }
   if (deprecationInfo.type === EasBuildDeprecationInfoType.Internal) {
-    Log.warn('This command is using API that soon will be deprecated, please update eas-cli.');
+    Log.warn('This command is using API that soon will be deprecated. Update EAS CLI.');
```
Comment (kbrandwijk) [inline packages/eas-cli/src/build/utils/printBuildInfo.ts:102]: Above, we use 'upgrade' vs 'update'.

**Review by kbrandwijk — COMMENTED** [review]
Few small nits for consistency (mostly stemming from the original wording) that just happened to be a bit more noticeable seeing all these changes listed closely together.

**Review by wkozyra95 — APPROVED** [review]
btw errors from https://github.com/expo/eas-build/blob/main/packages/eas-build-job/src/errors.ts are also printed in cli output, so it would be good to unify them.

**Review by dsokal — APPROVED** [review]

**Review by byCedric — APPROVED** [review]

**Review by brentvatne — APPROVED** [review]
thank you please

**Review by kbrandwijk — APPROVED** [review]

**Pull request merged**